django_feature_flip/helpers.py

```diff
@@ -28,7 +28,7 @@ def enabled(self, feature_name, actor=None):
         if feature.totally_enabled:
             return True
         elif actor:
-            return self.feature_enabled_for_a_group_of_actor(feature, actor)
+            return self.feature_enabled_for_a_group_of_actor(feature, actor) or feature.has_actor(actor.flip_id())
```

Comment:
Maybe you can move the whole condition to a new method and reduce the tree decision to one line like `return feature.totally_enabled or enable_for_actor(feature, actor)`. wdyt?